config/manager/manager.yaml

```diff
@@ -13,6 +13,14 @@ spec:
         env:
         - name: CRUNCHY_DEBUG
           value: "true"
+        - name: RELATED_IMAGE_POSTGRES_13
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-ha:centos8-13.3-0"
+        - name: RELATED_IMAGE_POSTGRES_13_GIS_3.1
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-gis-ha:centos8-13.3-3.1-0"
+        - name: RELATED_IMAGE_PGBACKREST
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:centos8-2.33-0"
+        - name: RELATED_IMAGE_PGBOUNCER
+          value: "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:centos8-1.15-0"
```

Comment:
📝 Recent Kustomize (v3, I think?) (in `kubectl` v1.21) can be configured to replace these using the [images](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/images/) kustomization.